Feature/docs (#27)
68publishers/consent-management-platform: README.md

  * [Installation](#installation)
* [Product Documentation](#product-documentation)
* [Development Guide](#development-guide)

> * [License](#license)

Reviewer: ditto

## About CMP

> The CMP is a standalone application for logging cookie consent and managing the cookies themselves on any website.

Reviewer: ```suggestion
The CMP is a standalone application for storing users' cookie consent and managing the cookies themselves on any website or application.
```

> You can implement the integration with the application in the cookie bar on your website, however, if you use [our solution](https://github.com/68publishers/cookie-consent), the integration is ready for you.

Reviewer: ```suggestion
```

We can refer to the cookie widget later.

![CMP Application](docs/images/dashboard.png)

## Getting Started

### Prerequisites

> - Docker Compose

Reviewer: ```suggestion
- Docker
```

Just Docker will be fine since Docker Compose tool is part of docker.
